internal/website/content/howto/runtimervar/runtimevar.md

```diff
@@ -0,0 +1,108 @@
+---
```

Comment:
This could all be placed into _index.md.

Reply:
Good point. I was thinking we could add more stuff later, but maybe YAGNI